### ISSUE TITLE
with_mask_rcnn 为True时，训练过程的问题

### ISSUE THREAD

**Issue by yangyang117** [drpredict/drpredict#1]
你好，我在看代码时，发下如下一段代码：
![图片](https://user-images.githubusercontent.com/49703383/123539826-332e9f80-d76e-11eb-8774-e17f1fe16e2d.png)
其中：
x2 = x2.fill_(0) 或 x2 = torch.zeros(x.size(0), 256, dtype=x.dtype, device=x.device)
将相应的数据都变为了0，没有使用到 mask rcnn 模型提出的特征？
